client/styles/Home.module.css

```diff
@@ -0,0 +1,121 @@
+.container {
+  min-height: 100vh;
+  padding: 0 0.5rem;
+  display: flex;
```

Comment:
i notice this file is in `client/styles` and being referenced by `client/pages`

does that mean all css files will be in a separate subdirectory than components?

Reply:
Yes. Next js is using css modules out of the box and we can alias `styles` directory if need be. I prefer styles to be in its own file tbh but most likely we will not need any of this if we fully commit to tailwind.

Reply:
tailwind (=